v2/pink-sb/src/lib/card/Button.svelte

```diff
@@ -21,6 +22,7 @@
     on:mouseup
     type="button"
     class:selected
+    aria-disabled={disabled}
```

Comment:
also needs {disabled}, buttons have that state 👍🏻

Reply:
done.